csrc/pf_io.c

```diff
@@ -221,5 +221,13 @@ FileStream *sdDeleteFile( const char *FileName )
     TOUCH(FileName);
     return NULL;
 }
+
+int sdRenameFile( const char *OldName, const char *NewName )
+{
+    UNIMPLEMENTED("sdRenameFile");
+    TOUCH(OldName);
+    TOUCH(NewName);
+    return -1;
```

Comment:
This returns -1 and the other unimplemented functions return 0.
Is -1 an error result?
Maybe the other ones should not have returned 0...